build/utils/yarn.go

```diff
@@ -233,7 +242,7 @@ func buildYarnV2DependencyMap(packageInfo *PackageInfo, responseStr string) (dep
 	return
 }
 
-// runYarnInfoOrList depends on the yarn version currently operating on the project, runs the command that gets the dependencies of the project
+// Depends on the yarn version currently operating on the project, runs the command that gets the dependencies of the project
```

Comment:
I recommend a grammar checker, to make sure the comments have correct grammar.
Let's fix the above comment.